service/dap/server_test.go

```diff
@@ -6546,11 +6567,29 @@ func runTestWithDebugger(t *testing.T, dbg *debugger.Debugger, test func(c *dapt
 	client.ExpectTerminatedEvent(t)
 
 	<-serverStopped
+	if pid > 0 {
+		syscall.Kill(pid, syscall.SIGKILL)
```

Comment:
syscall.Kill doesn't exist on windows, you have to use os.(*Process).Kill

Reply:
Ah, thanks for the pointer! Let me fix this.

Reply:
It doesn't matter, it still needs to compile.